test/actions/katello/host/applicable_errata_install_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'katello_test_helper'
+
+module ::Actions::Katello::Host::Erratum
+  class TestBase < ActiveSupport::TestCase
+    include Dynflow::Testing
+    include Support::Actions::Fixtures
+
+    class Installtest < TestBase
+      let(:action_class) { ::Actions::Katello::Host::Erratum::ApplicableErrataInstall }
+
+      let(:uuid) { 'uuid' }
+      let(:content_facet) { katello_content_facets(:content_facet_one) }
+      let(:host) do
```

Comment:
I thought we had a fixture for a host with a content facet (seems like I was wrong?). If not, this will do to avoid unnecessary mocking: `@host = FactoryBot.create(:host, :with_content)`

Reply:
Seems like attempting to use the factory for the host begins a cascade of dependencies for content views, etc.

Reply:
Sounds like what you have is simpler and probably more appropriate for this case then. Let's leave it. Thanks for trying